extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -132,6 +134,14 @@ private TenantConfigContext createTenantContext(Vertx vertx, OidcTenantConfig oi
             options.setClientSecretParameterName(null);
         }
 
+        OidcTenantConfig.Proxy proxyConfig = oidcConfig.getProxy();
+        if (proxyConfig.host.isPresent()) {
+            if (!proxyConfig.port.isPresent())
```

Comment:
This also requires `{` and `}` per the checkstyle rules AFAIK.

Reply:
Yeah, I have it. Probably it existed before formatting was run. Now it exists.